2009-05-23  Aurelien Jarno <aurelien@aurel32.net>

	* config.h.in (__KFREEBSD_KERNEL_VERSION): Add.

---
 config.h.in |    3 +++
 1 file changed, 3 insertions(+)

Index: glibc-2.28/config.h.in
===================================================================
--- glibc-2.28.orig/config.h.in
+++ glibc-2.28/config.h.in
@@ -109,6 +109,9 @@
 /* Linux specific: minimum supported kernel version.  */
 #undef	__LINUX_KERNEL_VERSION
 
+/* kFreeBSD specific: minimum supported kernel version.  */
+#undef	__KFREEBSD_KERNEL_VERSION
+
 /* Override abi-tags ABI version if necessary.  */
 #undef  __ABI_TAG_VERSION
 
